source/admin_manual/list_of_events.rst

```diff
@@ -1028,6 +1043,32 @@ Mail
 | uid                 | Mail IMAP UID number                                 |
 +---------------------+------------------------------------------------------+
 
+.. _event_mail_opened:
+
+mail_opened
+^^^^^^^^^^^
+
+.. versionadded:: 2.3.16
+
+A mail is opened.
+
++---------------------+--------------------------------------------------------+
+| Field               | Description                                            |
++=====================+========================================================+
+| reason              | Reason why the mail was opened. (optional, this field  |
+|                     | is added if opening reason is specified by the         |
+|                     | component.                                             |
```

Comment:
I think could be simplified to just `Reason why the mail was opened. (optional)`